psycop/common/feature_generation/application_modules/generate_feature_set.py

```diff
@@ -21,8 +33,53 @@
 
 log = logging.getLogger()
 
+ValueSpecification: TypeAlias = Union[
+    PredictorSpec, OutcomeSpec, BooleanOutcomeSpec, TimeDeltaSpec, StaticSpec
+]
+
 
 def generate_feature_set(
+    project_info: ProjectInfo,
+    eligible_prediction_times_frame: PredictionTimeFrame,
+    feature_specs: Sequence[ValueSpecification],
+    feature_set_name: str,
+    n_workers: int,
+    do_dataset_description: bool,
+    compute_lazily: bool = False,
+) -> None:
+    feature_set_dir = project_info.flattened_dataset_dir / feature_set_name
+    if feature_set_dir.exists():
+        while True:
```

Comment:
Because copy pasting is easy 🤷‍♂️ but the behaviour is kinda nice - waiting for user input. But I guess it's not necessary when using `input`? unsure - never really use it